msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md

```diff
@@ -125,7 +143,7 @@ media.getMedia((error: microsoftTeams.SdkError, blob: Blob) => {
 });
 ```
 
-**Calling `viewImages`  API by ID**
+**Calling `viewImages` API by ID returned by `selectMedia` API**:
 
 ```javascript
 view images by id:
```

Comment:
First two lines should be comments. Could you please add // in front of first two lines in this snippet?